docs/BinSkimRules.md

```diff
@@ -914,3 +914,25 @@ This rule emits CSV data to the console for every compiler/language/version comb
 
 ---
 
+## Rule `BA6001.DisableIncrementalLinking`
+
+### Description
+
+Incremental linking support increases binary size and can reduce runtime performance. Fully optimized release builds should not specify incremental linking.
+
+### Messages
+
+#### `Pass`: Pass
+
+'{0}' was compiled with incremental linking disabled.
+
+#### `Warning`: Warning
+
+'{0}' was compiled with incremental linking enabled, which inflates binary size and can reduce performance.
+
+#### `NonReleaseBuild`: Pass
+
+'{0}' was not compiled as release, skipping performance checks
```

Comment:
>was [](http://example.com/codeflow?start=6&length=3)

'{0}' appears to be compiled as release and is therefore not subject to performance checks.